### PR TITLE
update images

### PR DESCRIPTION
**Pull request by YanZhiwei**

### CODE REVIEW

File: doc/developtools/project_package.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Project Package
+# Package Project
 
-In Visual Studio Code, press "Ctrl+Shift+P" to show the Command Palette, and enter "Clicknium" to select and show "Pack". Press "Enter", and select the path where the Exe file of the packed project is stored according to the pop-up window.
+In Visual Studio Code, press "Ctrl+Shift+P" to show the Command Palette, and enter "Clicknium" to select and show "Package Project". Press "Enter", and select the path where the Exe file of the packed project is stored according to the pop-up window.
```
Comment (yongjuntian) [inline doc/developtools/project_package.md:3]: executable

File: doc/developtools/project_package.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-# Project Package
+# Package Project
```
Comment (yongjuntian) [inline doc/developtools/project_package.md:1]: how user  change the logo

File: doc/developtools/project_management.md [hunk 3]
```diff
@@ -35,22 +35,20 @@ In Visual Studio Code, press "Ctrl+Shift+P" to show the Command Palette, and ent
...
 4. **.gitignore**：When using Git, you can add or remove files that you want to ignore.
-   
-   
 
 # Run/Debug Project
 
 ## Run the project
 
-In Visual Studio Code, press "Ctrl+Shift+P" to show the Command Palette, and enter "Clicknium" to select and show " Run". When you press "Enter" to run the project, Clicknium extension will deploy the project running process based on app.yaml
+In Visual Studio Code, press "Ctrl+Shift+P" to show the Command Palette, and enter "Clicknium" to select and show " Run Project". When you press "Enter" to run the project, Clicknium extension will deploy the project running process based on app.yaml
```
Comment (robinbob) [inline doc/developtools/project_management.md:43]: In Visual Studio Code, press "Ctrl+Shift+P" to show the Command Palette, input or select "Clicknium: Run Project". The clicknium extension will deploy and run the project based on app.yaml.

File: doc/developtools/project_management.md [hunk 4]
```diff
@@ -35,22 +35,20 @@ In Visual Studio Code, press "Ctrl+Shift+P" to show the Command Palette, and ent
...
 ![project run](../img/run_project.gif) 
 
 ## Debug the project
 
 ### Basics
 
 In Visual Studio Code, set a breakpoint on the right side of the code editing window at first,
```
Comment (robinbob) [inline doc/developtools/project_management.md:50]: In Visual Studio Code, set a breakpoint to the code where you want to pause,

File: doc/developtools/project_management.md [hunk 5]
```diff
@@ -35,22 +35,20 @@ In Visual Studio Code, press "Ctrl+Shift+P" to show the Command Palette, and ent
...
-press "Ctrl+Shift+P" to show the Command Palette, and enter "Clicknium" to select and show " Debug". When you press "Enter" to run the project, Visual Studio Code will be debuging with the debug button shown at the top. 
+press "Ctrl+Shift+P" to show the Command Palette, and enter "Clicknium" to select and show " Debug Project". When you press "Enter" to run the project, Visual Studio Code will be debuging with the debug button shown at the top.
```
Comment (robinbob) [inline doc/developtools/project_management.md:51]: press "Ctrl+Shift+P" to show the Command Palette, input or select "Clicknium: Debug Project". The clicknium extension will start projec debugging with debug buttons shown at the top of Visual Studio Code.

File: doc/developtools/project_management.md [hunk 6]
```diff
@@ -10,10 +10,10 @@ When the project is created, a pop-up window in the lower right corner shows the
 
 ![project structure](../img/create_project_1.png)
 
-In Visual Studio Code, press "Ctrl+Shift+P" to show the Command Palette, and enter "Clicknium" to select and show " Run". When you press "Enter" to run the project, Clicknium extension will deploy the project running process based on app.yaml.
+In Visual Studio Code, press "Ctrl+Shift+P" to show the Command Palette, input or select "Clicknium: Run Project". The clicknium extension will deploy and run the project based on app.yaml.
 
 1. **app.py** an automation scripts file where the main function is the entry function for runing the project.
```
Comment (robinbob) [inline doc/developtools/project_management.md:15]: a python file where the main function is the entry for running the project.

File: doc/developtools/project_management.md [hunk 7]
```diff
@@ -10,10 +10,10 @@ When the project is created, a pop-up window in the lower right corner shows the
...
-   ![project appyy](../img/create_project_apppy.png)
+![project appyy](../img/create_project_apppy.png)
 
 2. **app.yaml** configuration files where you can configure the project, Python version, Python package, project entry file, cloud element repository, etc.
```
Comment (robinbob) [inline doc/developtools/project_management.md:18]: configuration file where you can configure the project, python version, python packages, project entry file and locator store references, etc.

File: doc/developtools/project_package.md [hunk 8]
```diff
@@ -1,12 +1,12 @@
...
 
-In Visual Studio Code, press "Ctrl+Shift+P" to show the Command Palette, and enter "Clicknium" to select and show "Pack". Press "Enter", and select the path where the Exe file of the packed project is stored according to the pop-up window.
-![](D:\Repos\ClickCorp\clicknium-docs\doc\img\pack_project.gif)
+In Visual Studio Code, press "Ctrl+Shift+P" to show the Command Palette, and enter "Clicknium" to select and show "Package Project". Press "Enter", and select the path to save the executable file.
```
Comment (robinbob) [inline doc/developtools/project_package.md:3]: ..., input or select "Clicknium: Package Project", then select the path to save the executable file.

**Review by robinbob — APPROVED** [review]

**Pull request merged**